xl_auth/templates/users/view.html

```diff
@@ -0,0 +1,85 @@
+<!-- users/view.html -->
+{% extends "layout.html" %}
+{% block content %}
+    <h1>{{ _('View User \'%(email)s\'', email=user.email) }}</h1>
+    <br/>
+    <div class="panel panel-default">
+        <dl class="dl-horizontal">
+            <dt>{{ _('Full name') }}:</dt>
+            <dd>{{ user.full_name }}</dd>
+
+            <dt>{{ _('Last Login At') }}:</dt>
+            <dd>{{ '-' if not user.last_login_at else user.last_login_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}</dd>
+
+            <dt>{{ _('Active Account') }}:</dt>
+            <dd>{{ _('Yes') if user.is_active else _('No') }}</dd>
+
+            <dt>{{ _('Administrator') }}:</dt>
+            <dd>{{ _('Yes') if user.is_admin else _('No') }}</dd>
+
+            <dt>{{ _('Cataloguing Administrator') }}:</dt>
+            <dd>{{ _('Yes') if user.is_cataloging_admin else _('No') }}</dd>
+
+            <dt>{{ _('Last Modified') }}:</dt>
+            <dd>
+                {{ user.modified_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}, {{  _('by') }}
+                <a href="{{ url_for('user.view', user_id=user.modified_by.id) }}">
+                    {{ user.modified_by.full_name }}
+                </a>
+            </dd>
+
+            <dt>{{ _('Created At') }}:</dt>
+            <dd>
+                {{ user.created_at.strftime('%Y-%m-%dT%H:%M:%SZ') }}, {{  _('by') }}
+                <a href="{{ url_for('user.view', user_id=user.created_by.id) }}">
+                    {{ user.created_by.full_name }}
+                </a>
+            </dd>
+        </dl>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <strong>{{ _('Permissions') }}</strong>
```

Comment:
@mxtthias Again, here we should different the table description to not be misguiding, e.g. 
* [ ] ``_('Permissions')`` when viewing as superuser
* [ ] ``_('Permissions')`` when viewing as the user itself
* [ ] ``_('Cataloging Admin Permissions')`` when viewing as a user that does not have ``cataloging_admin`` permissions on any of the collections that the user has access to (thus only seeing the cataloging admin subset)
* [ ] ``_('Subset of Permissions')`` when viewing as a user that DOES have ``cataloging_admin`` permissions on one of the collections that the user has access to (thus seeing a union of the *cataloging admin* and *anything on collection X* subsets)

Reply:
For the third point, do we mean that if a regular user (user A) views a user (user B), A will only see B's cataloging admin permissions? That sounds reasonable. Unless we should prevent regular users from viewing users altogether?

For the fourth point, do we mean that a cataloging admin (A) will see only the user permissions that are for the collections that A is admin for? E.g. user A is admin for A1, A2, and B2 and user B has permissions for A2, B1, and B2. User A viewing user B would only see permissions for A2 and B2.